### PR TITLE
FEATURE: F1-266 support drilling

### PR DESCRIPTION
**Pull request by hackerstanislav**
JIRA: F1-266

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18702/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20393/ : SUCCESS in 10m 32s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18706/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20397/ : SUCCESS in 9m 33s

File: libs/sdk-ui-charts/src/charts/repeater/Repeater.tsx [hunk 1]
```diff
@@ -56,7 +58,17 @@ export interface IRepeaterBucketProps {
 /**
  * @beta
  */
-export interface IRepeaterProps extends IBucketChartProps, IRepeaterBucketProps {}
+export interface IRepeaterProps extends IBucketChartProps, IRepeaterBucketProps {
+    /**
+     * Configure drillability; e.g. which parts of the visualization can be interacted with.
```
Comment (kandl) [inline libs/sdk-ui-charts/src/charts/repeater/Repeater.tsx:63]: Should we mention current limitations? (only attributes in columns are supported now). Maybe at least in the documentation? + We should update docs  - add drill props.

**Review by kandl — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18712/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20406/ : SUCCESS in 8m 16s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/2077/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20408/ : SUCCESS in 9m 32s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1877/ : SUCCESS in 10m 36s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1859/ : SUCCESS in 7m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5373/ : SUCCESS in 7s

**Pull request merged**